sources/csv/README.md

```diff
@@ -0,0 +1,114 @@
+# HarbourBridge: CSV-to-Spanner Migration
+
+HarbourBridge is a stand-alone open source tool for Cloud Spanner evaluation 
+and migration. We now support loading data from CSVs. This assumes a Spanner
+database with Schema already exists and HarbourBridge loads the data for you.
+
+
+To run HarbourBridge to migrate CSVs, it should be used only with the data 
```

Comment:
Might sound better if we say

"To migrate CSVs via HarbourBridge, provide a manifest file and run it with the data subcommand"
And then describe the manifest file